lib/fathead/mdn_css/output.txt

```diff
@@ -544,44 +774,61 @@ box-orient	A										<section class="prog__container"><p>The CSS box-orient pro
 cm	A										<section class="prog__container"><p>One centimeter (10 millimeters).</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/length#cm
-color	A			css properties							<section class="prog__container"><p>The color property sets the foreground color of an element's text content, and its decorations. It doesn't affect any other characteristic of the element; it should really be called text-color and would have been named so, save for historical reasons and its appearance in CSS Level 1.</p><p><b>Initial Value: </b><em>Varies from one browser to another</em></p><pre><code>/* A CSS Level 1 color */\ncolor: red;\n\n/* The only color added in CSS Level 2 (Revision 1) */\ncolor: orange;\n\n/* CSS Level 3 color, sometimes called a SVG or X11 color */\ncolor: antiquewhite;\n\n/* The color 'lime' with the 3-character dash notation */\ncolor: #0f0;\n\n/* The color 'lime' with the 6-character dash notation */\ncolor: #00ff00;\n\n/* A color using the available functional notations */\ncolor: rgba( 34, 12, 64, 0.3);\n\n/* Use the color of this element's direct ancestor */\ncolor: currentcolor;\n\n/* Global values */\ncolor: inherit;\ncolor: initial;\ncolor: unset;\n</code></pre></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/color
+codepoint range	A										<section class="prog__container"><p>A range of Unicode code points. So for example, U+0025-00FF means include all characters in the range U+0025 to U+00FF.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/unicode-range#Values
+collapse	A										<section class="prog__container"><p>For table rows, columns, column groups, and row groups the row(s) or column(s) are hidden and the space they would have occupied is removed (as if display: none were applied to the column/row of the table). However, the size of other rows and columns is still calculated as though the cells in the collapsed row(s) or column(s) are present. This was designed for fast removal of a row/column from a table without having to recalculate widths and heights for every portion of the table. For XUL elements, the computed size of the element is always zero, regardless of other styles that would normally affect the size, although margins still take effect. For other elements, collapse is treated the same as hidden.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/visibility#Values
+color	A			css properties							<section class="prog__container"><p>Is a <color> denoting the color to apply to the edges.</p></section>	https://developer.mozilla.org/en-US/docs/Web/CSS/border-color#Values
```

Comment:
@hchienjo ok looks like we should definitely ignore the `#Values` URLs, in this case we're over-writing the CSS Color attribute.

If we find we want to keep the `#Values` content, we may need to prefix them with the attribute name to help ensure they're unique. I can imagine a user might search "css border-color color"?